database/buildSchema/22_application_response.sql

```diff
@@ -1,10 +1,16 @@
 -- application response
+CREATE TYPE public.application_response_status AS ENUM (
+    'Draft',
+    'Submitted'
+);
 
 CREATE TABLE public.application_response (
-    id serial primary key,
-    template_element_id integer references public.template_element(id),
-    application_id integer references public.application(id),
+    id serial PRIMARY KEY,
+    template_element_id integer REFERENCES public.template_element (id),
+    application_id integer REFERENCES public.application (id),
+    "status" public.application_response_status DEFAULT 'Draft',
```

Comment:
Pretty sure "status" doesn't need the quotes (it doesn't have it in "review_assignment" for example).

Reply:
9647e30ce59ee7b0568b2a56f41e4d778b932548